### ISSUE TITLE
Support Graph Copy to local target

### ISSUE THREAD

**Issue by sajayantony** [oras-project/oras-go#43]
As a part of the discoverers works, the oras project is defining the work to support graph copy from on remote registry to another. 
The copy API semantics is about targets and if local is a target this doesn't work. 

For e.g something like this 
```
oras copy --recursive myregistry/net-monitor:v1 <local_target>
```

**Comment by sajayantony** [comment]
@shizhMSFT - Is there an issue for removing dependency on containerd here or will you open the issue as a design?

**Comment by shizhMSFT** [comment]
I will open an new design issue for discussion.

**Comment by shizhMSFT** [comment]
Resolved by #66 and tracked in #49 .
